metrics/ttfp/run-benchmark.jl

```diff
@@ -64,69 +153,33 @@ end
 
 project1 = make_project_folder("current-pr")
 Pkg.activate(project1)
-Pkg.develop([(; path="./MakieCore"), (; path="."), (; path="./CairoMakie")])
-Pkg.precompile()
+pkgs = [(; path="./MakieCore"), (; path="."), (; path="./$Package"), (;name="BenchmarkTools")]
+Package == "WGLMakie" && push!(pkgs, (; name="ElectronDisplay"))
+Pkg.develop(pkgs)
+precompile_pr = @elapsed Pkg.precompile()

```

Comment:
wouldn't it already be precompiled after the `develop` call?

Reply:
I guess it might. But I'm not benchmarking that anymore anyways.